acid/features/status/controller.py

```diff
@@ -18,7 +18,13 @@ def show_status(pipename=None):
     url = service.status_endpoint(zuul_url, zuul_endpoint)
     resource = service.fetch_json_data(endpoint=url)
     queues = service.make_queues(resource['pipelines'], pipename, zuul_url)
-    return render_template('status.html', queues=queues, pipename=pipename)
+    refs_list = []
+    for queue in queues:
+        for buildset in queue.buildsets:
+            refs_list.append("collapse" + buildset.ref)
```

Comment:
did you consider to use list comprehensions?

Reply:
yes I consider

Reply:
```refs_list = ["collapse" + buildset.ref for queue in queues 
                                       for buildset in queue.buildsets]```

Reply:
Lets push it further
```python
refs_list = [f'collapse{b.ref}' for q in queues for b in q.buildsets]
`